src/sonic-device-data/tests/platformJson_checker

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+try:
+    import re
+    import sys
+    import glob
+    import json
+except ImportError as e:
+    raise ImportError (str(e) + "- required module not found")
+try:
+  basestring
+except NameError:
+  basestring = str
+
+# Global variable
+PORT_ATTRIBUTES = ["index", "lanes", "alias_at_lanes", "breakout_modes", "default_brkout_mode"]
+ATTR_LEN = len(PORT_ATTRIBUTES)
+
+def usage():
+    print "Usage: " + sys.argv[0] + " <platform_json_file>"
+    sys.exit(1)
+
+
+def check_port_attr(port_attr):
+    for each_key in port_attr:
+        if each_key not in PORT_ATTRIBUTES:
+            print "Error: "+ each_key + " is not the correct Port attribute."
+            return False
+        if not port_attr[each_key]:
+            print "Error: "+ each_key + " has no value."
+            return False
+        if not isinstance(port_attr[each_key], basestring):
+            print "Error:value type of  "+ each_key + " must be string."
+            return False
+    return True
+
+def check_file(platform_json_file):
+    try:
+        platform_cap_file = open(platform_json_file,"r")
+        platform_file_data = platform_cap_file.read()
+        port_dict = json.loads(platform_file_data)
+
+        for each_port in port_dict:
+
+            # Validate port at top level
+            port_id = re.search("Ethernet(\d+)", each_port)
```

Comment:
Define the "Ethernet" at the beginning as we may change it in the future.

Reply:
addressed